infrastructures/infrastructure-openstack/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/OpenstackInfrastructure.java

```diff
@@ -46,37 +46,52 @@
 
     private static final Logger logger = Logger.getLogger(OpenstackInfrastructure.class);
 
-    @Configurable(description = "The Openstack_Username")
+    @Configurable(description = "Openstack username")
     protected String username = null;
 
-    @Configurable(description = "The Openstack_Password")
+    @Configurable(description = "Openstack password")
     protected String password = null;
 
-    @Configurable(description = "The Openstack_EndPoint")
+    @Configurable(description = "Openstack user domain")
+    protected String domain = null;
+
+    @Configurable(description = "Openstack identity endPoint")
     protected String endpoint = null;
 
-    @Configurable(description = "Resource manager hostname or ip address")
-    protected String rmHostname = generateDefaultRMHostname();
+    @Configurable(description = "Openstack scope prefix")
+    protected String scopePrefix = null;
 
-    @Configurable(description = "Connector-iaas URL")
-    protected String connectorIaasURL = "http://" + generateDefaultRMHostname() + ":8080/connector-iaas";
+    @Configurable(description = "Openstack scope value")
+    protected String scopeValue = null;
 
-    @Configurable(description = "Image")
+    @Configurable(description = "Openstack region")
+    protected String region = null;
+
+    @Configurable(description = "Openstack identity version")
+    protected String identityVersion = null;
+
+    @Configurable(description = "Openstack image")
     protected String image = null;
 
     @Configurable(description = "Flavor type of OpenStack")
-    protected int flavor = 3;
+    protected String flavor = "3";
```

Comment:
should the value "3" be extracted as constant?

Reply:
"3" removed. Thanks